gadgets/trace_open/test/trace_open_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"flag"
+	"io/fs"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/inspektor-gadget/inspektor-gadget/integration"
```

Comment:
The idea is to use only the `testing` pkg and not import things from `integration`. So, we also need to move the [containers pkg](https://github.com/inspektor-gadget/inspektor-gadget/pull/2666) within the `testing` pkg (not actually move but copy for now but the same you did for `command` and `match`).